src/modules/30-secrets/components/CreateUpdateSecret/__tests__/CreateUpdateSecret.test.tsx

```diff
@@ -23,6 +23,7 @@ jest.mock('services/cd-ng', () => ({
   usePostSecret: jest.fn().mockImplementation(() => ({ mutate: jest.fn() })),
   usePostSecretFileV2: jest.fn().mockImplementation(() => ({ mutate: jest.fn() })),
   usePutSecretFileV2: jest.fn().mockImplementation(() => ({ mutate: jest.fn() })),
+  useGetGcpRegions: jest.fn().mockImplementation(() => ({ mutate: jest.fn() })),
```

Comment:
THis works fine, but you can simplify with `mockReturnValue`
```suggestion
  useGetGcpRegions: jest.fn().mockReturnValue({ mutate: jest.fn() }),
```